crates/red_knot_python_semantic/src/types.rs

```diff
@@ -1208,17 +1208,36 @@ impl<'db> Type<'db> {
                 })
             }
 
-            Type::Instance(InstanceType { class, .. }) => {
+            instance_ty @ Type::Instance(InstanceType { .. }) => {
                 // Since `__call__` is a dunder, we need to access it as an attribute on the class
                 // rather than the instance (matching runtime semantics).
```

Comment:
I think this comment can be removed, since this method is no longer handling this; it's part of the semantics of `call_dunder`